qiime2/sdk/usage.py

```diff
@@ -738,69 +739,67 @@ class DiagnosticUsage(Usage):
     """
     def __init__(self):
         super().__init__()
-        self.recorder = []
```

Comment:
This was leftover from the design process - the out-of-band recorder shouldn't be necessary - Scope should handle everything.